From 63631e451866680da28cd11a8f57e5b53960dbaf Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 25 Jul 2018 10:11:41 +0100
Subject: [PATCH 19/20] config/common: Clean up debug packages

 * Include all debug packages via PRODUCT_PACKAGES_DEBUG.

 * Remove oprofiled and sqlite3 from the list, because the former
   no longer exists and the latter is already part of
   build/target/product/core_base.mk. Also remove gdbserver, since
   it's already built by default.

Change-Id: Icde54061cd5a85d381b756364287344c3d216957
---
 config/common.mk | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index fa227286..30870af5 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -173,16 +173,11 @@ PRODUCT_PACKAGES += \
     bash \
     bzip2 \
     curl \
-    gdbserver \
     htop \
     lib7z \
     libsepol \
-    micro_bench \
-    oprofiled \
     pigz \
     powertop \
-    sqlite3 \
-    strace \
     unrar \
     unzip \
     vim \
@@ -233,12 +228,14 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     media.recorder.show_manufacturer_and_model=true
 
 # These packages are excluded from user builds
-ifneq ($(TARGET_BUILD_VARIANT),user)
-PRODUCT_PACKAGES += \
+PRODUCT_PACKAGES_DEBUG += \
+    micro_bench \
     procmem \
-    procrank
+    procrank \
+    strace
 
 # Conditionally build in su
+ifneq ($(TARGET_BUILD_VARIANT),user)
 ifeq ($(WITH_SU),true)
 PRODUCT_PACKAGES += \
     su
-- 
2.17.1

